plugins/scaffolder-annotator-action/package.json

```diff
@@ -43,8 +39,6 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@backstage/backend-common": "^0.23.3",
-    "@backstage/backend-dynamic-feature-service": "^0.2.15",
     "@backstage/backend-plugin-api": "^0.7.0",
     "@backstage/catalog-model": "^1.5.0",
```

Comment:
This dependency looks unused. I think you can remove it.

Reply:
You are right. Thank you for spotting! I don't know why I assumed that it is being used since it creates catalog entities, although not being used in code. Updated.